src/dearpygui.cpp

```diff
@@ -544,12 +554,15 @@ PyInit__dearpygui(void)
 	MV_ADD_COMMAND(clear_selected_links);
 
 	// plots
-	MV_ADD_COMMAND(is_plot_queried);
-	MV_ADD_COMMAND(get_plot_query_area);
```

Comment:
`get_plot_query_area`: We can also keep this function as a wrapper in `_deprecated.py`. It should extract the first rect from `get_plot_query_rects` and return it. If `get_plot_query_rects` returns an empty list, `get_plot_query_area` can return `(0, 0, 0, 0)` - this won't match the old behavior but otherwise should be safe enough. And of course tell the user to use `get_plot_query_rects` instead (via `@deprecated`).